### DISCUSSION TITLE
Monitoring S.M.A.R.T

### DISCUSSION THREAD

**Discussion by Ulrar** [prometheus-operator/kube-prometheus#2306] [Q&A] [answered]
Hi,

Apologies if I missed it, but I don't see any kind of S.M.A.R.T monitoring included in this.
Any advice on what to use that could integrate in this nicely for anyone running bare metal ?

Thank you

**Comment by simonpasquier** [comment, marked as answer]
you should look at https://github.com/prometheus-community/node-exporter-textfile-collector-scripts

**Reply by Ulrar** [reply]
Do you mean to try to get the status from somewhere ?
Not sure how that works, I usually use smartctl but I suppose it reads that from proc somewhere indeed
